addon/ng2/utilities/get-dependent-files.ts

```diff
@@ -0,0 +1,105 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as ts from 'typescript';
+
+interface MapType {
```

Comment:
I was doing exactly the same thing. So, do you want me to just change `MapType` to `ImportMap` even though this mapping will a `string => string[]` only?